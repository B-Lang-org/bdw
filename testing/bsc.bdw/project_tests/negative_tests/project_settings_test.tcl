
# Opens the specified project
handlers::open_project project.bspec

# Adds mult4 dir to search paths
handlers::set_search_paths

# Adds mult4 dir to search paths
handlers::set_search_paths mult5

# Adds mult4 dir to search paths
handlers::set_search_paths negative_tests.exp

# Adds mult4 dir to search paths
handlers::set_search_paths mult4

#  
handlers::set_project_editor 

# Sets the project editor to emacs
handlers::set_project_editor vedit

# Sets the project editor to emacs
handlers::set_project_editor gvim -path editor 

# Sets the project editor to emacs
handlers::set_project_editor emacs -options ee -options www 

# Sets the top file and module 
handlers::set_top_file 

# Closes the project
handlers::close_project

# Exits the workstation
exit
