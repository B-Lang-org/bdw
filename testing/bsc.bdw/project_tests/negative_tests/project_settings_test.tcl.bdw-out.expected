The search path is not specified
Search path entry 'mult5' is not  a directory.
Search path entry 'negative_tests.exp' is not  a directory.
The editor name is not specified
Wrong editor name. 
The editor name can be  gvim, emacs, other
Unknown option: -path
Unknown option: -options
The top file is not specified
